lib/roo_on_rails/railties/database.rb

```diff
@@ -3,7 +3,7 @@ module Railties
     class Database < Rails::Railtie
       initializer 'roo_on_rails.database', after: 'active_record.initialize_database' do
         ActiveSupport.on_load :active_record do
-          $stderr.puts 'initializer roo_on_rails.database'
+          Rails.logger.debug 'initializer roo_on_rails.database'
```

Comment:
Might be nice to change these to something like `initializer: 'roo_on_rails.database'` now we can ;-)

Reply:
Ooh, good idea. The `msg` is still mandatory though, guess I'll have it say just `"loading"`.